taskcluster/ci/browsertime/kind.yml

```diff
@@ -44,10 +44,10 @@ job-defaults:
         platform:
             by-abi:
                 arm64-v8a: android-hw-p2-8-0-android-aarch64-shippable-qr/opt
-                armeabi-v7a: android-hw-g5-7-0-arm7-api-16-shippable-qr/opt
+                armeabi-v7a: android-hw-a51-11-0-arm7-shippable/opt
```

Comment:
@92kns you'll need to add the `-qr` to match this platform: https://searchfox.org/mozilla-central/source/taskcluster/ci/test/test-platforms.yml#372

See these tests in mozilla-central: https://treeherder.mozilla.org/jobs?repo=mozilla-central&searchStr=android%2Ca51%2Cqr&revision=09bcd3a0096b6d38671c7da6c6980dd9a7e373bb